pkg/controller/controller.go

```diff
@@ -72,7 +72,9 @@ func (c *Controller) Start() error {
 	}
 
 	c.client = NewXdsClient(c.mode, c.bpfWorkloadObj)
-	c.client.workloadController.Processor.Sm = secertManager
+	if c.client.workloadController != nil {
+		c.client.workloadController.Processor.Sm = secertManager
```

Comment:
The PR also contains a bug fix. Supplement the description in the PR title and description.